ci/jenkins/docker-images.ini

```diff
@@ -18,7 +18,7 @@
 # This data file is read during when Jenkins runs job to determine docker images.
 [jenkins]
 ci_arm: tlcpack/ci-arm:20230615-060132-62a5e7acf
-ci_cortexm: tlcpack/ci-cortexm:20230613-060122-21361a63a
+ci_cortexm: tlcpackstaging/ci_cortexm:20230710-060128-a60cd0fec
```

Comment:
IIRC it's possible to keep the image name as `tlcpack/ci-cortexm` but specify the tlcpackstaging tag: `20230710-060128-a60cd0fec`. If so, it seems better to do that for consistency. cc @leandron who may know more

Reply:
Thanks @lhutton1! I replaced tag with tlcpack/ci-cortexm.